### DISCUSSION TITLE
图片扫描件，OCR识别部分行的最后一个字没有框全，识别不到

### DISCUSSION THREAD

**Discussion by gongdj** [PaddlePaddle/PaddleOCR#13314] [Q&A]
python tools/infer/predict_system.py --image_dir="D:\ocr\scan-2.png" --det_model_dir="F:\github\ocr_model\ch_PP-OCRv4_det_server_infer"  --rec_model_dir="F:\github\ocr_model\ch_PP-OCRv4_rec_server_infer"

检测出来，发现 识别的结果为：
[2024/07/09 09:57:53] ppocr DEBUG: 合同结算总金额的（%为质保金，质量保修期满后由乙方提出书面申请.经甲方3(, 0.965

正确的应该是：
合同结算总金额的（%为质保金，质量保修期满后由乙方提出书面申请.经甲方30
![scan-2](https://github.com/PaddlePaddle/PaddleOCR/assets/10403160/714845fc-e14e-45d6-883a-86c4425c67a1)

具体识别：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/10403160/f2f26a87-6c97-4363-b548-e7f9a7cf5b1e)

**Comment by GreatV** [comment]
换成最新版本试试看

**Reply by gongdj** [reply]
已经是新版本的：
(paddle_env) PS F:\github\PaddleOCR> pip list
Package                Version
---------------------- -----------
anyio                  4.4.0
astor                  0.8.1
attrdict               2.0.1
Babel                  2.15.0
bce-python-sdk         0.9.14
beautifulsoup4         4.12.3
blinker                1.8.2
cachetools             5.3.3
certifi                2024.6.2
charset-normalizer     3.3.2
click                  8.1.7
colorama               0.4.6
contourpy              1.1.1
cssselect              1.2.0
cssutils               2.11.1
cycler                 0.12.1
Cython                 3.0.10
decorator              5.1.1
et-xmlfile             1.1.0
exceptiongroup         1.2.1
fire                   0.6.0
Flask                  3.0.3
flask-babel            4.0.0
fonttools              4.53.0
future                 1.0.0
h11                    0.14.0
httpcore               1.0.5
httpx                  0.27.0
idna                   3.7
imageio                2.34.1
imgaug                 0.4.0
importlib_metadata     7.1.0
importlib_resources    6.4.0
itsdangerous           2.2.0
Jinja2                 3.1.4
kiwisolver             1.4.5
lazy_loader            0.4
lmdb                   1.4.1
lxml                   5.2.2
MarkupSafe             2.1.5
matplotlib             3.7.5
more-itertools         10.2.0
networkx               3.1
numpy                  1.24.4
opencv-contrib-python  4.6.0.66
opencv-python          4.6.0.66
opencv-python-headless 4.10.0.82
openpyxl               3.1.3
opt-einsum             3.3.0
packaging              24.0
paddleocr              2.8.0
paddlepaddle           2.6.1
pandas                 2.0.3
pdf2docx               0.5.8
pillow                 10.3.0
pip                    24.0
premailer              3.10.0
protobuf               3.20.2
psutil                 5.9.8
pyclipper              1.3.0.post5
pycryptodome           3.20.0
PyMuPDF                1.24.5
PyMuPDFb               1.24.3
pyparsing              3.1.2
python-dateutil        2.9.0.post0
python-docx            1.1.2
pytz                   2024.1
PyWavelets             1.4.1
PyYAML                 6.0.1
rapidfuzz              3.9.3
rarfile                4.2
requests               2.32.3
scikit-image           0.21.0
scipy                  1.10.1
setuptools             69.5.1
shapely                2.0.4
six                    1.16.0
sniffio                1.3.1
soupsieve              2.5
termcolor              2.4.0
tifffile               2023.7.10
tqdm                   4.66.4
typing_extensions      4.12.2
tzdata                 2024.1
urllib3                2.2.1
visualdl               2.5.3
Werkzeug               3.0.3
wheel                  0.43.0
zipp                   3.19.2

**Reply by GreatV** [reply]
`det_db_unclip_ratio` 设大点试试

**Reply by gongdj** [reply]
感谢大佬，这个好使，由1.5调整到2.0后能识别了，多谢！
